### ISSUE TITLE
Improve Mailvelope Composer widget UI

### ISSUE THREAD

**Issue by bnvk** [mailvelope/mailvelope#299]
Currently, the Mailvelope inline composer widget (which shows up in your webmail) looks like this:

![screen shot 2015-04-06 at 4 13 48 pm](https://cloud.githubusercontent.com/assets/48677/7005402/fa3f85e2-dc77-11e4-9c0a-537529e2d382.png)

I propose the following improvements in the widget that can be seen in a rendering below:

![mailvelope better widget](https://cloud.githubusercontent.com/assets/48677/7005736/25f8c1e6-dc7b-11e4-943e-93d2b2ef32c5.png)

**Add "Mailvelope" title to widget** 

I believe this better communicates to user that the widget is part of the Mailvelope plugin and makes it better standout for this reason

**Add "Trash" button / icon in addition to undo** 

Currently, the "undo" button is slightly confusing. Before I understood what it was, I accidentally deleted an encrypted message blog I had just inserted in the composer.

That said, I think keeping the "undo" functionality makes sense if a user has pasted in more than one PGP encrypted blob as then the undo actually goes back in history and "un-does" things. Thus, I propose the "trash" icon if there is only 1 item in "history" but the trash + undo if there is more than one item in the history.

**Comment by jancborchardt** [comment]
I’d suggest adding text to the icons to remove the confusion entirely. (Btw heya @bnvk ;D)

**Comment by uggrock** [comment]
I find that these suggestions are great and believe that adding text to the icons as proposed by @jancborchardt or even adding tooltips with some more explanations will solve the current usability issue.

**Comment by jancborchardt** [comment]
@uggrock oh man, you’re involved with Mailvelope as well? Then we’ll definitely need to sit together as @bnvk will probably visit me in Munich next week. :)

**Comment by bnvk** [comment]
I agree 100% about adding text as well. I can flesh out this design overall a bit more if the changes are welcome and desired- perhaps something that communicates relating to encryption & security as opposed to just normal email style icons!

Thanks for chiming in @jancborchardt ;) Yes, let's hang in Munich, even if I see @uggrock this weekend in Frankfurt!

**Comment by rugk** [comment]
:+1: for this idea.

FYI tooltips were suggested in https://github.com/mailvelope/mailvelope/issues/121
